test/markup/fsharp/operators.expect.txt

```diff
@@ -1,28 +1,77 @@
+
+<span class="hljs-keyword">let</span> <span class="hljs-keyword">inline</span> (<span class="hljs-operator">++</span>) (x<span class="hljs-operator">:</span> <span class="hljs-symbol">&#x27;Monoid</span>) (y<span class="hljs-operator">:</span> <span class="hljs-symbol">&#x27;Monoid</span>) <span class="hljs-operator">:</span> <span class="hljs-symbol">&#x27;Monoid</span> <span class="hljs-operator">=</span> Plus.Invoke x y
+
+<span class="hljs-keyword">let</span> <span class="hljs-keyword">inline</span> (<span class="hljs-operator">&lt;|&gt;</span>) (x<span class="hljs-operator">:</span> <span class="hljs-symbol">&#x27;``Functor&lt;&#x27;T&gt;``</span>) (y<span class="hljs-operator">:</span> <span class="hljs-symbol">&#x27;``Functor&lt;&#x27;T&gt;``</span>) <span class="hljs-operator">:</span> <span class="hljs-symbol">&#x27;``Functor&lt;&#x27;T&gt;``</span> <span class="hljs-operator">=</span> Append.Invoke x y
+
+<span class="hljs-keyword">let</span> <span class="hljs-keyword">inline</span> (<span class="hljs-operator">=&gt;&gt;</span>) (s<span class="hljs-operator">:</span> <span class="hljs-symbol">&#x27;``Comonad&lt;&#x27;T&gt;``</span>) (g<span class="hljs-operator">:</span> <span class="hljs-symbol">&#x27;``Comonad&lt;&#x27;T&gt;``</span><span class="hljs-operator">-&gt;</span><span class="hljs-symbol">&#x27;U</span>) <span class="hljs-operator">:</span> <span class="hljs-symbol">&#x27;``Comonad&lt;&#x27;U&gt;``</span> <span class="hljs-operator">=</span> Extend.Invoke g s
+
+<span class="hljs-comment">// Code quotation</span>
 
 <span class="hljs-keyword">open</span> Microsoft.FSharp.Quotations
 <span class="hljs-comment">// A typed code quotation.</span>
-<span class="hljs-keyword">let</span> expr : Expr&lt;<span class="hljs-type">int</span>&gt; = &lt;@ <span class="hljs-number">1</span> + <span class="hljs-number">1</span> @&gt;
+<span class="hljs-keyword">let</span> expr <span class="hljs-operator">:</span> Expr<span class="hljs-operator">&lt;</span><span class="hljs-type">int</span><span class="hljs-operator">&gt;</span> <span class="hljs-operator">=</span> <span class="hljs-operator">&lt;@</span> 1 <span class="hljs-operator">+</span> 1 <span class="hljs-operator">@&gt;</span>
```

Comment:
What happened to `number` here getting lost?

Reply:
Oops! Well spotted, this is a little embarrassing...

This should be fixed now.